endpoints/openrtb2/auction_test.go

```diff
@@ -121,45 +99,49 @@ func TestJsonSampleRequests(t *testing.T) {
 			"currency-conversion/custom-rates/errors",
 		},
 	}
+
```

Comment:
In the PR description, you mention the exporting of the `AdaptedBidder` interface so that a mock bidder adapter to mock bidder server map can be built in this file. I'm having trouble finding this map, could you point it out to me?

Reply:
Originally `buildTestEndpoint(test testCase, paramValidator openrtb_ext.BidderParamValidator)` was written into `endpoints/openrtb2/auction_test.go` but in an effort to refactor said file, this function was moved to `endpoints/openrtb2/test_utils.go`. Exported `AdaptedBidder` is instantiated in line 1084 of `endpoints/openrtb2/test_utils.go` shown below:
```
1078 func buildTestEndpoint(test testCase, paramValidator openrtb_ext.BidderParamValidator) (httprouter.Handle, *httptest.Server, *httptest.Server, *httptest.Server, *httptest.Server, error) {
1079     bidderInfos := getBidderInfos(test.Config.getAdaptersConfigMap(), openrtb_ext.CoreBidderNames())
1080     bidderMap := exchange.GetActiveBidders(bidderInfos)
1081     disabledBidders := exchange.GetDisabledBiddersErrorMessages(bidderInfos)
1082
1083     // Adapter map with mock adapters needed to run JSON test cases
1084     adapterMap := make(map[openrtb_ext.BidderName]exchange.AdaptedBidder, 0)
1085
1086     // AppNexus mock bid server and adapter
1087     appNexusBidder := mockBidderHandler{bidInfo: test.Config.MockBidder, bidderName: "appnexus"}
endpoints/openrtb2/test_utils.go
```